xml/html_title.xml

```diff
@@ -331,4 +331,16 @@
     <param pos="0" name="os.product" value="Linux"/>
     <param pos="0" name="os.cpe23" value="cpe:/o:debian:debian_linux:-"/>
   </fingerprint>
+  <fingerprint pattern="^Eagle Eye Network">
+    <description>EagleEyeNetworks to manage IoT cameras home webpage</description>
+    <example>Eagle Eye Networks (Local)</example>
+    <param pos="0" name="hw.product" value="Eagle Eye Bridge"/>
```

Comment:
I believe the product is called just Bridge.  See https://www.een.com/product/bridge-models/, where in the tables near the bottom they call the devices Bridge or CMVR.